deploy/kubernetes/rbac.yaml

```diff
@@ -87,6 +87,19 @@ rules:
 - apiGroups: ["coordination.k8s.io"]
   resources: ["leases"]
   verbs: ["get", "watch", "list", "delete", "update", "create"]
+- apiGroups: ["storage.k8s.io"]
```

Comment:
Add a note that this is only needed if you're using the storage capacity feature